### PR TITLE
Added multiple resource links

### PR DESCRIPTION
**Pull request by EdCharbeneau**

### CODE REVIEW

**Comment by guardrex** [comment]
I made minor modifications to shorten the first one (btw - *Great web app design ... gorgeous site!*). Let me know if you don't like the changes, and we can discuss it further.

File: community.md [hunk 1]
```diff
@@ -143,6 +145,7 @@ Below is a list of projects and resources from the Blazor community (links in ea
 
 * [Blazor First Look on LinkedIn Learning](https://www.linkedin.com/learning/blazor-first-look)
 * [Free Blazor Training Course](https://www.devexpress.com/support/training/blazor/)
+* [Blazor StateHasChanged: The weekly community live show](https://www.youtube.com/edwardcharbeneau)
```
Comment (danroth27) [inline community.md]: This isn't really a course. It could go under Other, or maybe we should a new section for Streams / Podcasts?

**Review by danroth27 — CHANGES_REQUESTED** [review]

File: community.md (hunk 1 above)
Comment (guardrex) [reply community.md]: :heart: "Streams and Podcasts"

I'll move it to that. Stand-by.

**Review by danroth27 — APPROVED** [review]
:shipit:

**Comment by EdCharbeneau** [comment]
@danroth27 Where did the community page end up after the new website went live?

**Comment by guardrex** [comment]
Going with Awesome Blazor repo. Our page is gone 👋 *bye* 👋.

https://github.com/AdrienTorris/awesome-blazor

**Comment by danroth27** [comment]
@edcharbeneau https://blazor.net/community now redirects to Awesome Blazor, and we link to there from the new home page. We felt it was better to let the community page be driven by the community. Let me know If you have any questions or concerns about this.

**Pull request merged**